examples/1d/elixir_advection_basic.jl

```diff
@@ -8,62 +8,52 @@ using Trixi
 advectionvelocity = 1.0
 equations = LinearScalarAdvectionEquation1D(advectionvelocity)
 
-initial_condition = initial_condition_convergence_test
+# Create DG solver with polynomial degree = 3 and Lax-Friedrichs flux as surface flux
```

Comment:
```suggestion
# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
```
To add some more keywords that might be familiar to more people... This change could also be applied to the other files.

Reply:
Fixed in c71513eb.